exporters/trace/gcp_exporter/internal/recordable_test.cc

```diff
@@ -16,12 +16,30 @@ TEST(Recordable, TestSetAttribute)
     const common::AttributeValue bool_value = true;
     rec.SetAttribute(bool_key, std::move(bool_value));
 
-    // Set 'integer' type
+    // Set 'int' type
     const nostd::string_view int_key = "int_key";
-    const int64_t seven = 7;
-    const common::AttributeValue int_value = seven;
+    const int seven_int = 7;
+    const common::AttributeValue int_value = seven_int;
     rec.SetAttribute(int_key, std::move(int_value));
 
+    // Set 'int64_t' type
+    const nostd::string_view int64_t_key = "int64_t_key";
```

Comment:
You may consider using templates here to reduce code duplication (see [OTLP Recordable test](https://github.com/open-telemetry/opentelemetry-cpp/blob/60f6920acb94089c903aaa9c822c7cbb7125554c/exporters/otlp/test/recordable_test.cc#L197)). It's totally optional though.

Reply:
Yes, I was considering that. Just trying to contain all the different types( string, bool as well as all int types) in one single test.

@IlyaKobelevskiy @ziweizhao thoughts?